src/main/java/io/confluent/connect/jdbc/gp/gpload/GPLoadDataIngestionService.java

```diff
@@ -189,6 +189,7 @@ private void loadPendingFiles(String gploadBinary) {
 
     }
     private void loadFile(String gploadBinary, File yamlFile, File csvFile, File logFile) throws Exception{
+        if (!config.gpOnLoadedData) return;
```

Comment:
change config name to skip.data.load
use this condition before caling this method